<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Eyes Rig Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="leftEyesGeo_le"/>
      </item>
      <item>
       <widget class="QPushButton" name="leftEyesGeo_btn">
        <property name="text">
         <string>Left Eye Geo</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="rightEyesGeo_le"/>
      </item>
      <item>
       <widget class="QPushButton" name="rightEyesGeo_btn">
        <property name="text">
         <string>Right Eye Geo</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Distance Control</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="distance_control_sb">
        <property name="maximum">
         <double>9999.989999999999782</double>
        </property>
        <property name="value">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="create_btn">
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
